src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CcdCaseRetriever.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@Service
+public class CcdCaseRetriever {
+    public static final String CASE_TYPE_ID = "Bulk_Scanned";
```

Comment:
You won't be able to write unless you have definition with above case type id imported in AAT